packages/mui-material-nextjs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/material-nextjs",
-  "version": "5.15.11",
+  "version": "6.0.0-alpha.0",
```

Comment:
This was the only package that needed updates after https://github.com/mui/material-ui/pull/41648